eth_defi/utils.py

```diff
@@ -1,6 +1,13 @@
 """Bunch of random utilities."""
 import socket
 
+# Assert approximate integer
+def approx(actual :float , expected : float, percentage_threshold : float) -> bool:
```

Comment:
This function is not needed. You can use `pytest.approx` 

Example: https://github.com/tradingstrategy-ai/web3-ethereum-defi/blob/e2d8265ee599bcb58fa62dddacf6899bcf2a9247/tests/test_uniswap_v2_pair.py#L199